Classes/Aspects/NodeTypeSchemaCacheHeaderAspect.php

```diff
@@ -1,49 +0,0 @@
-<?php
```

Comment:
Wait, why is this removed? As I see, it's currently fixed in Neos 4.0, so Neos 3.3. would still be affected by the lack of caching, no?

Reply:
Well, but this is targeted to master, which requires Neos 4.0. Do I miss something?

Reply:
I'm blind